compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/MemberTypeDescriptorNode.java

```diff
@@ -78,7 +78,7 @@ public MemberTypeDescriptorNodeModifier modify() {
     /**
      * This is a generated tree node modifier utility.
      *
-     * @since 2.0.0
+     * @since 2201.3.0
```

Comment:
```suggestion
     * @since 2201.4.0
```

Update wherever applicable.

Reply:
Not addressed.